imgui-SFML.cpp

```diff
@@ -207,15 +216,28 @@ struct WindowContext {
 #endif
 #endif
 
+#ifdef VIEWPORTS_ENABLE
```

Comment:
Ah, I see, the `WindowContext` constructor might become ambiguous because of default arguments... You shouldn't make them default them to avoid ambiguity then, in my opinion.